ibm_catalog.json

```diff
@@ -633,18 +623,14 @@
               "version": "v3.0.7"
             },
             {
-              "name": "deploy-arch-ibm-observability",
-              "description": "Enable to provision and configure IBM Cloud Logs, Cloud Monitoring, Metrics routing and Activity Tracker event routing for analysing logs and metrics generated by the Event Notifications instance.",
+              "name": "deploy-arch-ibm-cloud-logs",
+              "description": "Enable to provision and configure IBM Cloud Logs for analysing logs generated by the Event Notifications instance.",
```

Comment:
As discussed in one of the DA Rallies, description should start with -
`Provision and configure...` OR `Configure IBM Cloud Logs...` instead of `Enable to configure...`

It applies for others Monitoring and AT description as well.